src/OAuth2/Server.php

```diff
@@ -1068,6 +1265,25 @@ protected function createDefaultIdTokenTokenResponseType()
         return new IdTokenToken($this->getAccessTokenResponseType(), $this->getIdTokenResponseType());
     }
 
+    /**
+     *
```

Comment:
Komischer Docblock

Reply:
der ist aufgebaut, wie an den anderen Stellen in der Klasse